tools/libvirt_test.sh

```diff
@@ -95,6 +95,12 @@ ARTIFACTS="${ARTIFACTS:-/tmp/artifacts}"
 SSH_DATA_DIR=$(/usr/libexec/osbuild-composer-test/gen-ssh.sh)
 SSH_KEY=${SSH_DATA_DIR}/id_rsa
 
+VIRT_LOG="$ARTIFACTS/libvirt_tests-virt-install-console.log"
+if [ ! -f "$VIRT_LOG" ]; then
```

Comment:
nitpick: This isn't needed, `touch` just updates timestamps if the file already exists and returns 0.